a/ansible-lint/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "ansible-lint",
+    "github_url": "https://github.com/ansible/ansible-lint",
+    "version": "main",
```

Comment:
This should be latest version for which build script is tested. Currency jobs expect to run on release branches/tags

Reply:
got it , forgot to change here , checking lastest version  in build script